core/lib/src/config/builder.rs

```diff
@@ -220,16 +220,20 @@ impl ConfigBuilder {
     /// ```rust
     /// use rocket::config::{Config, Environment};
     ///
+    /// let none: Option<String> = None;
```

Comment:
This should probably be called `cert_store_path` or something like that for clarity.